src/charm.py

```diff
@@ -123,6 +123,11 @@ def __init__(self, *args):
             charm=self,
             source_type="alertmanager",
             source_url=self._external_url,
+            refresh_event=[
+                self.ingress.on.ready,
+                self.ingress.on.revoked,
+                self.server_cert.on.cert_changed,
+            ],
```

Comment:
In the [other PR](https://github.com/canonical/prometheus-k8s-operator/pull/632/files), you're also doing it on update-status: should we have it also in the list here?

Reply:
Good point :)